packages/app-store/office365calendar/_metadata.ts

```diff
@@ -3,10 +3,10 @@ import type { AppMeta } from "@calcom/types/App";
 import _package from "./package.json";
 
 export const metadata = {
-  name: "Office 365 / Outlook.com Calendar",
+  name: "Outlook Calendar",
   description: _package.description,
   type: "office365_calendar",
-  title: "Office 365 / Outlook.com Calendar",
+  title: "Outlook Calendar",
```

Comment:
I propose shortening the app name to just "Outlook Calendar"

Reply:
This should be another PR but I'll allow it